state_management/javascript/http/order-processor/index.js

```diff
@@ -1,42 +1,40 @@
-import axios from "axios";
+import axios from "axios"
 
-const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
-const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
-const DAPR_STATE_STORE = 'statestore'
+const DAPR_HOST = process.env.DAPR_HOST ?? "http://localhost"
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT ?? "3500"
+const DAPR_STATE_STORE_NAME = 'statestore'
+const stateStoreBaseUrl = `${DAPR_HOST}:${DAPR_HTTP_PORT}/v1.0/state/${DAPR_STATE_STORE_NAME}`
 
 async function main() {
-  for(var i = 1; i <= 10; i++) {
-    const orderId = i;
-    const order = {orderId: orderId}; 
-    const state = [{
-      key: orderId.toString(),
-      value: order
-    }];
+  // For each loop, Save order, Get order, and Delete order
+  for (var i = 1; i <= 10; i++) {
```

Comment:
would want to use `let` instead of `var` since the latter is global scoped

Reply:
Changed to `let`